samples/nrf9160/modem_shell/Kconfig

```
@@ -0,0 +1,98 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "Kconfig.zephyr"
+
+menu "Nordic Modem Shell config"
+
+config MOSH_IPERF3
+	bool "Enable iperf3 support"
```

Also an enable here.
```suggestion
	bool "iPerf3 support"
```

Also it seems the right way to write it is iPerf3, https://iperf.fr/

---

I was just changing these and thought of removing the word "support" also.

---

That would also be fine with me.